README.md

```diff
@@ -123,6 +146,107 @@ To enable this configuration use the `extends` property in your
 }
 ```
 
+## Configuration
+
+There are some configuration options available that will be shared across all the plugin rules. This is achieved using [ESLint Shared Settings](https://eslint.org/docs/user-guide/configuring/configuration-files#adding-shared-settings). These shared settings are meant to be used if you need to restrict the Aggressive Reporting mechanism, **so please before configuring any of these settings**, read more about [`eslint-plugin-testing-library` Aggressive Reporting mechanism](docs/migrating-to-v4-guide.md#aggressive-reporting), and [how it's affected by these settings](docs/migrating-to-v4-guide.md#shared-settings).
+
+If you are sure about configuring the settings, these are the options available:
```

Comment:
Does it make sense to put all of this in the main README given that it's not something that most users would need? Maybe a separate page that explains this in detail and this document can just contain the summary and explanation of what can be configured and why someone might need to configure it. Probably doesn't need to go into such detail about how to actually do it.

Reply:
Yeah, I think I'm gonna move this Configuration section to the very bottom of the README, put this detailed explanation into migrating guide, and leave just the "how to" here, referencing the detailed version.